src/cashier/currency.es6

```diff
@@ -4,7 +4,7 @@ import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import html from 'text!cashier/currency.html';
 
-require(['text!cashier/currency.html']);
+require(['common/util']);
```

Comment:
That seems to work too, although I couldn't find any similar statement in the documentation.